static/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Critter</title>
+    <link rel="stylesheet" href="/static/index.css">
+</head>
+<body>
+    <header>
+        <nav class="header-flex">
+            <img src="" alt="profile picture">
+            <h1>Home</h1>
+            <form action="post">
+                <label for="create-crit">Send a Crit!</label>
```

Comment:
The "for" attribute of the label is for the _id_ of the element it is a label for. Because this is a label for your input, you must include a "create-crit" `id` on that element.